test/functional/test_runner.py

```diff
@@ -225,8 +225,7 @@
     'feature_rbf.py --descriptors',
     'mempool_packages.py',
     'mempool_package_onemore.py',
-    'rpc_createmultisig.py --legacy-wallet',
```

Comment:
Unless I'm mistaken you do want both tests run (--legacy-wallet and --descriptors). We are still supporting the legacy wallet and hence we should still test it. I think you've taken it out because of the discussion in #24635 but the legacy wallet is [not yet deprecated](https://github.com/bitcoin/bitcoin/issues/20160).

Reply:
In this particular test, the wallet, when compiled is used for `addmultisigaddress` which descriptor wallet doesn't actually support, so I don't think there is any difference in running with or without the flag.
The legacy wallet when compiled is actually used even without the flag so there is no need to run the test twice.